clojure-editors/editor-user-guides/vscode-calva.md

```diff
@@ -47,10 +47,10 @@ Once you have a running REPL, use these commands to help you develop your code.
 
 | Evaluate code             | Keybinding         | Description                                                          |
 |---------------------------|--------------------|----------------------------------------------------------------------|
-| Namespace/file            | `Ctrl+Alt+c ENTER` | Loead the current file/namespace and any namespaces it depends on    |
```

Comment:
This breaks the table formatting unfortunately.

Reply:
Can you please elaborate how this breaks the table formatting. I think I just removed a character. Is it introducing something unintentional?